data/json/effects_on_condition/npc_eocs/appearance_eocs.json

```diff
@@ -0,0 +1,325 @@
+[
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_hair_growth_tracking",
+    "//": "Hair usually grows at a rate of 1 cm per month.  This is not a hard number, because hair growth is very complex.",
+    "//2": "For the purposes of this EOC right now, we'll assume this rate.",
+    "//3": "For ease of calculation, all math will be done with centimeters for hair growth.",
+    "//4": "For reference starting point, a buzzcut is 0.32 centimeters long.",
+    "//5": "1 inch is 2.54 centimeters.",
+    "recurrence": { "math": [ "(648000000 - ((u_val('health') * 180000) ) )" ] },
```

Comment:
did you mixed up numbers here? even with 200 health, the result would be 612000000, which is 7083 days
i suspect you put two zeroes more than it should be (recurrence smallest value is 1 second, not 1 move)

Reply:
I likely misplaced some zeroes, yes. I'll see about having that fixed.